README.adoc

```diff
@@ -15,25 +15,55 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+* This is a relationship tracker based on a desktop Address Book application. It has a GUI but most of the user interactions
+happen using a CLI (Command Line Interface).
+* It is primarily targeted at *private investigators* who wish to find a way to maintain their ever-growing and ever-changing
+information base easily.
+
+== Why Intelli?
+
+Wondering why you should use Intelli? +
+Here are three extra-special features Intelli offers that may convince you why:
+
+* Add named relationships between your contacts
+
+  Intelli enables you to easily keep track of who's who in vast and complicated social networks
+
+* Add and edit confidence estimates to relationships between contacts
+
+  Worried about getting the relationship between A and B wrong? Not quite sure
+  that A and B are indeed longstanding nemeses? As your relationship tracker, Intelli
+  will help to remind you of the uncertainty surrounding a particular relationship,
+  so that you can update it in the future when you gain more valuable information
+
+* Visualise detailed relationships in a graph
+
+  Sometimes, even the best private investigators can get entangled in the complex web
+  of relationships which they are investigating. However, with Intelli you can obtain a graphic
+  overview of these relationships, allowing you to see named relationships coupled with
+  confidence estimates so that you no longer have to fear keeping track of the messy networks
+  among your contacts
+
+Still doubting why? Feel free to browse the full catalogue of features in Intelli by
+visiting the <<UserGuide#, user guide>>!
 
 == Site Map
 
 * <<UserGuide#, User Guide>>
 * <<DeveloperGuide#, Developer Guide>>
-* <<LearningOutcomes#, Learning Outcomes>>
```

Comment:
I think we can go ahead and delete `LearningOutcomes.adoc` and `TestDoucment.adoc` as well (if we are removing links to them anyway).